src/mugs.js

```diff
@@ -480,7 +480,7 @@ define([
             return this.parentMug && this.parentMug.isInRepeat();
         },
         supportsRichText: function() {
-            return this.options.richText && this.form.useRichText !== false;
+            return this.options.richText && this.form.noRichText === false;
```

Comment:
> Its possible that when loading an empty form it won't have a value for noRichText. You should be able to fix that by initializing it in Form. and then you would be able to get rid of the strict checks

Didn't see this. https://github.com/dimagi/Vellum/pull/667/commits/91ce41479d43620eb5e1260bd927ff91a714e54a feels not quite complete, because it's not checking for some form property being passed in...but it seems to work.